src/components/atoms/CircleLink/CircleLink.tsx

```diff
@@ -0,0 +1,14 @@
+import styles from './CircleLink.module.scss';
+
+interface Props {
+  link: string;
+  icon: JSX.Element;
+}
+
+export default function CircleLink({ link, icon }: Props) {
```

Comment:
What do you think about using arrow functions (AFs) for React components instead? From my opinion it is more preferable way to declare functional components